source/extensions/tracers/zipkin/zipkin_tracer_impl.cc

```diff
@@ -35,6 +35,10 @@ void ZipkinSpan::log(SystemTime timestamp, const std::string& event) {
   span_.log(timestamp, event);
 }
 
+void ZipkinSpan::setBaggage(absl::string_view, absl::string_view) {}
```

Comment:
nit: can you add the TODO for this being unimplemented here as well?

Reply:
fixed in https://github.com/envoyproxy/envoy/pull/12260/commits/05dbaa50f5cc45f851bf881120f849f2d4e5e80f